configuration/download_cache.rst

```diff
@@ -0,0 +1,82 @@
+.. _download_cache:
+
+Download cache
+==============
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+.. warning::
+
+    This is an **advanced** feature, use it only if you fully understand this document. Do not use it if you are mixing api V1 (without revisions)
+    and api V2 (with revisions) clients.
+
+
+
+Conan implements a shared download cache that can be used to reduce the time needed to populate the Conan package cache
+with commands like :command:`install`, :command:`create`.
+
+This cache is purely an optimization mechanism. It is completely different to the Conan package cache, (typically the ``<userhome>/.conan`` folder).
```

Comment:
Somewhere around here or the previous lines we need a link to the Conan cache: https://docs.conan.io/en/latest/mastering/custom_cache.html?highlight=conan%20cache

Reply:
Added link